requirements.txt

```diff
@@ -1,2 +1,3 @@
 docker-py
 dockerfile-parse
+#pygit2
```

Comment:
I just filed a PR for Python 2.6 compat, so let's wait and see if upstream accepts it: https://github.com/uniphil/venvgit2/pull/12